spec/system/jobseekers_can_search_for_jobs_spec.rb

```diff
@@ -10,7 +10,7 @@
 
     it "displays page 1 jobs" do
       within("ul.search-results") { expect(page).to have_css("li", count: 2) }
-      expect(page).to have_css(".search-results-sorting__stats", text: strip_tags(I18n.t("app.pagy_stats_html", from: 1, to: 2, total: 6, type: "results")))
+      expect(page).to have_css(".govuk-grid-column-full", text: strip_tags(I18n.t("app.pagy_stats_html", from: 1, to: 2, total: 6, type: "results")))
```

Comment:
Checking that something is inside `govuk-grid-column-full` is not useful as it's not a specific class. I'd just check the standard
`expect(page).to have_content strip_tag....`